buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy

```diff
@@ -24,6 +24,7 @@ import org.gradle.api.JavaVersion
 import org.gradle.api.Project
 import org.gradle.api.tasks.compile.JavaCompile
 import org.gradle.api.tasks.testing.Test
+import org.jetbrains.kotlin.gradle.internal.AndroidExtensionsExtension
```

Comment:
I am a little worried about relying on an internal class here. The `getByType` api uses reflection under the hood anyways. Can we please just do this instead?

- Remove the extension dependency in build.gradle
- `hasExperimentalKotlinAndroidExtensions = project.androidExtensions.experimental` in a try catch block guarded by `hasKotlinAndroidExtensions`

Reply:
Sure